helpers/chatbot/intentResponse.js

```diff
@@ -0,0 +1,99 @@
+// Function to manage custom intents are defined here
+module.exports = {
+    CheckMalariaSymptoms: function(parameters, done) {
+
+        const symptomScores = {
+            chills: 2,
+            fatigue: 2,
+            fever: 2,
+            malaise: 2,
+            shivering: 2,
+            sweating: 2,
+            headache: 1,
+            nausea: 1,
+            'pain in muscles': 1,
```

Comment:
Can we have _ or - here instead of a string?

Reply:
@Raxerz can you please elaborate?

Reply:
I mean can we use '_' or '-' instead of spaces as separators here. 
Apologies for not being clear earlier.

Reply:
Got it @Raxerz :+1:

Reply:
@lunayach Did you make this change?

Reply:
@Raxerz made changes where I removed hardcoded strings. Would push that change.

And this change may not be possible. Because this is something Dialogflow extracts form the user's statement. i.e. user says, "I have pain in muscles" and then in one of the parameters we store "pain in muscles" to infer later the parameter.